tests/unit/test_0Z_model_validation.py

```diff
@@ -49,6 +49,12 @@ def test_single_validation(
     if EXPORT_OUTPUT:
         save_json_data(f"{case_id}.json", result)
 
+    # test calculate with only node result
```

Comment:
I'd like to see a unit tests for more than one key.

(and add one for a set of componets as well)